### PR TITLE
Feat: add quote forms

### PR DESCRIPTION
**Pull request by josuesan**
## Purpose

Create the flow to get all policies that are soon to expiry and make new quotations with quote forms

## Solution Approach

- Create an endpoint to quote forms
- Update config files
- Add tests and validations
- Refactor database folder to providers folder
- Create Quote renovations provider
- Create functions and helpers to update IDs after quotation
- Add Logger to important parts of the flow

## Demo
image

### CODE REVIEW

File: src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    const response = await fetch(`${this.quoteFormsService}/${id}`);
+    return response.json();
+  }
+
+  async createQuote(quoteForm: QuoteDto): Promise<{ oldId: string; newId: string }> {
+    const oldId = quoteForm.id;
+    const newId = v4();
+    const body = JSON.stringify({ ...quoteForm, id: newId });
+
+    const response = await fetch(this.quoteFormsService, {
```
Comment (suarezcumare) [inline src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts:21]: you can use https://github.com/comparaonline/octopus-prime/blob/release/src/web-server/geo/communes/commune.service.ts#L7

mono client: https://github.com/comparaonline/mono-client

**Review by suarezcumare — CHANGES_REQUESTED** [review]

File: src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { Injectable } from '@nestjs/common';
+import * as config from 'config';
+import { v4 } from 'uuid';
+
+@Injectable()
+export class QuoteRenovationsRepositoryCl {
+  private quoteFormsService: string = config.get('services.quoteForms.cl');
+
+  async getQuoteById(id: string): Promise<QuoteDto> {
+    const response = await fetch(`${this.quoteFormsService}/${id}`);
```
Comment (suarezcumare) [inline src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts:12]: the same here

File: src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts (hunk 1 above)
Comment (josuesan) [reply src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts]: Done

File: src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts (hunk 2 above)
Comment (josuesan) [reply src/modules/renovations/infrastructure/providers/quote-renovations.repository.cl.ts]: Done

**Pull request merged**